src/Dialect/ONNX/Transforms/Recompose.cpp

```diff
@@ -31,6 +31,8 @@
 #include "src/Dialect/ONNX/ONNXOps/OpHelper.hpp"
 #include "src/Dialect/ONNX/ONNXOps/ShapeHelper.hpp"
 #include "src/Dialect/ONNX/Transforms/Recompose.hpp"
+
+#include "mlir/TableGen/Builder.h"
```

Comment:
this isn't an internal library, prefer using `<>` 
```suggestion
#include <mlir/TableGen/Builder.h>
```